build_exe.bat

```diff
@@ -0,0 +1,2 @@
+pyinstaller --onefile updateHostsFile.py
+copy /y dist\updateHostsFile.exe .
```

Comment:
Why aren't you moving instead of copy?

Reply:
Um... that's a good question and I don't have a good answer. I think I'm just more used to typing copy.

Reply:
Don't create duplicate like that, please...

Reply:
Gotcha!